src/components/Description/Description.jsx

```diff
@@ -1,12 +1,19 @@
 import { PropTypes } from "prop-types";
 import React from "react";
 import styles from "./styles.css";
+import { classNames } from "../../utils";
 
-const Description = ({ body }) =>
-  body ? <div className={styles.description}>{body}</div> : null;
+const Description = ({ body, style, className }) =>
+  body ? (
```

Comment:
```suggestion
  body && (
```

Reply:
Actually, this will throw an error if the body is not passed to Description. `Error: Description(...): Nothing was returned from render. This usually means a return statement is missing. Or, to render nothing, return null.` I'll keep returning null.

Reply:
oh, i see, makes sense

Reply:
why is body optional ? i think the consumer should wrap the description with boolean if want to hide it,
body should be required in this component as it doesn't make sense without it, what do you think ?

Reply:
I'd have to touch the Checkbox component, which is not relative to this PR. But can be done.